src/ReflectionProperty.php

```diff
@@ -190,43 +176,53 @@ public function getValue($object = null)
     }
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
      */
-    public function isDefault()
+    public function isDefault(): bool
     {
-        // TRUE if the property was declared at compile-time
+        $this->initializeInternalReflection();
```

Comment:
See my general comment - it should not be allowed for this library to load core reflection, as `goaop/framework` operates **before autoloader** and this is why if you will try to initialize reflection inside weaving process, you may end up with "class not exists" error :(

Reply:
So, please keep as much methods without internal reflection initialization